deployment.go

```diff
@@ -70,7 +69,7 @@ var (
 )
 
 type deploymentStore interface {
-	List() (deployments []v1beta1.Deployment, err error)
+	List() []interface{}
 }
```

Comment:
Or keep them as they were and add a wrapper around the cache's `List` method doing the typecast there. Saves us from dragging `interface{}` through the actual application logic layers and we can keep working with more idiomatic go.